make/buf/all.mk

```diff
@@ -112,13 +114,20 @@ bufgenerateclean:: bufgeneratecleango
 bufgenerateprotogo:
 	$(BUF_BIN) generate proto --template data/template/buf.go.gen.yaml
 
+.PHONY: bufgenerateprotoconnectgo
+bufgenerateprotoconnectgo: $(PROTOC_GEN_CONNECT_GO)
+	$(BUF_BIN) generate $(BUF_PROTO_SOURCE) --template data/template/buf.connect-go.gen.yaml
+	$(BUF_BIN) generate proto --template data/template/buf.connect-go.gen.yaml
+	$(BUF_BIN) generate --template data/template/buf.connect-go.gen.yaml --path envoy/service/auth/v3 buf.build/envoyproxy/envoy
```

Comment:
This shouldn't be required for the CLI, its specific to the schema registry. We might need to update the template files a bit.

Reply:
Specifically referring to the envoy auth service.